### DISCUSSION TITLE
check if the mods are all up to date before updating to the latest version

### DISCUSSION THREAD

**Discussion by Alddste** [itzg/docker-minecraft-server#2912] [Ideas]
In my sporadic search of the documentation, I haven't come across an option regarding having a server hold back at a certain version until the listed mods in MODRINTH_PROJECTS are all at the latest version, and then upgrade. This would fix my unmanned server. Is there an option, or would this be hard to implement?

**Comment by itzg** [comment]
That's probably doable somehow but as you suspected, definitely not easy. What do you mean exactly by "latest version"? The Minecraft version?

**Reply by Alddste** [reply]
Yes, the latest minecraft version. For example, version 1.21 was released, while mods such as debugify, no-chat-reports, and incendium are still on 1.20.6.

**Reply by itzg** [reply]
I could see that being some kind of "updates ready" subcommand of mc-image-helper that could be exec'ed on demand to let someone know there's the potential for a coordinated upgrade of mods/pkugins.
